sql/src/main/java/io/crate/executor/transport/executionphases/InterceptingRowReceiver.java

```diff
@@ -119,11 +123,13 @@ private void tryForwardResult(Throwable throwable) {
                 new KillJobsRequest(Collections.singletonList(jobId)), new ActionListener<KillResponse>() {
                     @Override
                     public void onResponse(KillResponse killResponse) {
+                        LOGGER.debug("Killed {} jobs before forwarding the failure", killResponse.numKilled());
                         rowReceiver.fail(failure);
                     }
 
                     @Override
                     public void onFailure(Throwable e) {
+                        LOGGER.warn("Failed to kill job, forwarding failure anyway..", e);
```

Comment:
could also be trace, since it is not a failure condition which is critical or where the user shoulld react on